src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -13,7 +13,7 @@ case class RawlsUserSubjectId(value: String) extends UserAuthType
 case class RawlsGroupName(value: String) extends UserAuthType
 case class RawlsGroupEmail(value: String) extends UserAuthType
 case class RawlsBillingProjectName(value: String) extends UserAuthType
-case class RawlsGroupMemberList(userEmails: Seq[String], subGroupEmails: Seq[String])
+case class RawlsGroupMemberList(userEmails: Option[Seq[String]] = None, subGroupEmails: Option[Seq[String]] = None, userSubjectIds: Option[Seq[String]] = None, subGroupNames: Option[Seq[String]] = None)
```

Comment:
ATM, @jmthibault79 is the only programatic customer and he will have only subject ids. Otherwise it is admin users and they will likely have only email addresses.